00-variables.tf

```
@@ -9,6 +9,29 @@ variable "cluster_id" {
   type        = number
 }
 
+variable "iam_instance_profile_controller" {
```

```suggestion
variable "iam_instance_profile_control_plane" {
```

I'd like to be more specific about where this IAM profile is used - for the control plane notes.